frontend/src/components/shared/WorkflowSamplesTable/SampleTableColumns.tsx

```diff
@@ -393,6 +458,8 @@ export const SAMPLE_NEXT_STEP_BY_STUDY_FILTER_KEYS: { [key in SampleColumnID]: s
 	[SampleColumnID.KIND]: 'sample_next_step__sample__derived_samples__sample_kind__name',
 	[SampleColumnID.NAME]: 'sample_next_step__sample__name',
 	[SampleColumnID.INDIVIDUAL]: 'sample_next_step__sample__derived_samples__biosample__individual__name',
+	[SampleColumnID.PARENT_CONTAINER]: 'sample_next_step__sample__container__location__barcode',
+	[SampleColumnID.PARENT_COORDINATES]: 'sample_next_step__sample__container__location__coordinate__name',
```

Comment:
This filter look wrong ... This should be the end of path as sample_next_step. ...sample__container__coordinate__name.